04/List.js

```diff
@@ -1,9 +1,23 @@
 import React from 'react';
+import { v4 as uuid } from 'uuid';
 
 class List extends React.Component {
-    render() {
-        return <ul>Lista plików</ul>
-    }
+	render() {
+		const { files } = this.props;
+		const newFiles = files.map(({ name, size, content }) => (
+			<li key={uuid()}>
```

Comment:
To rozwiązanie jest złe w tym miejscu. BO za każdym przeładowaniem (ponowny render) mamy inny `key` - tego typu dane należy definiować przy dodzwanianiu do `state`, a tu wykorzystywać. Wtedy ciągle mamy ten sam `uuid`.